README.md

```diff
@@ -405,26 +405,14 @@ All required libraries to run notebooks should be placed in `nbs/requirements.tx
 To build the maps, run
 
 ```
-> ./docker/build
-> ./docker/generate-maps
+> ./scripts/cibuild
 ```
 
-Now if you commit the `maps` folder, you'll make edits to the `covidcaremap.org/maps` sites.
-Make sure all data is placed correctly in the viz folders (i.e. run the notebook to [Process visualization data](notebooks#process-visualization-data).
+This will go through each map subdirectory and run its corresponding `cibuild` script. You can preview the visualizations locally by running `python -m http.server 9000` in the `_site` directory.
```

Comment:
I don't have jekyll installed locally so this step won't work without setup - is there an easy way to dockerize this build process?

Reply:
I wrapped the Netlify Docker build image in ba6bdd4af8665b3ba9e0150c75242df622c584ec with `./docker/build-website`. This should make it so that anyone with Docker installed can generate the website locally.